Mathlib/Geometry/Euclidean/Triangle.lean

```diff
@@ -264,6 +289,33 @@ theorem dist_sq_eq_dist_sq_add_dist_sq_sub_two_mul_dist_mul_dist_mul_cos_angle (
 
 alias law_cos := dist_sq_eq_dist_sq_add_dist_sq_sub_two_mul_dist_mul_dist_mul_cos_angle
 
+/-- **Law of sines** (sine rule), angle-at-point form. -/
+theorem sin_angle_mul_dist_eq_sin_angle_mul_dist {p₁ p₂ p₃ : P} :
```

Comment:
```suggestion
theorem sin_angle_mul_dist_eq_sin_angle_mul_dist (p₁ p₂ p₃ : P) :
```
which matches the theorem above

(this doesn't apply to the later theorems due to the non-degeneracy hypotheses)